docs/modules/databases/db2.md

```diff
@@ -13,12 +13,13 @@ Running DB2 as a stand-in for in a test:
 public class SomeTest {
 
     @ClassRule
-    public Db2Container db2 = new Db2Container()
-        .acceptLicense();
+    public TestcontainersRule<Db2Container> db2 = new TestcontainersRule<>(
```

Comment:
While you are touching this file, I suggest changing this field to be static (fields annotated with `@ClassRule` must be static; the code here would result in a test failure).